tests/pytorch/sparse/test_elementwise_op_sp.py

```diff
@@ -20,7 +20,9 @@ def all_close_sparse(A, row, col, val, shape):
     assert A.shape == shape
 
 
-@pytest.mark.parametrize("v_scalar", [2, 2.5])
+@pytest.mark.parametrize(
+    "v_scalar", [2, 2.5, torch.tensor(2), torch.tensor(2.5)]
```

Comment:
Do we expect to support things like `torch.tensor([2])`?

Reply:
I think so.  It allows us to backpropagate through the scalar coefficient at least.